analytics/population/nc_pop_recent.ipynb

```diff
@@ -0,0 +1,2597 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    def area_sq_miles(geo):
</pre><p>What is the type of <code>geo</code> here? Would be good to note.</p><p> </p><p>Also, please move this function definition closer to where it's being used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hackforla/311-data/pull/1336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Relocated the function closer to where it is being applied. 'geo' is the polygon geometry that includes a series of latitude and longitude coordinate pair that encloses an area, in this case each of the NC. &nbsp;</p>